zmsadmin/js/page/availabilityDay/index.js

```
@@ -101,35 +101,52 @@ class AvailabilityPage extends Component {
+                .filter((availability) => {
+                    return (
+                        (availability.__modified || 
+                        (availability.tempId && availability.tempId.includes('__temp__'))) &&
+                        !this.hasErrors(availability)
+                    );
+                })
+                .map(availability => {
+                    const sendAvailability = Object.assign({}, availability);
+                    if (availability.tempId) {
+                        delete sendAvailability.tempId;
+                    }
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)